Cannot Start h:  No module named h.server

Hi there, following the installation guide: https://h.readthedocs.org/en/latest/hacking/install.html

when trying to start the service via make dev I have been getting some errors:

(h) make dev
bash: gunicorn: command not found
make: *** [dev] Error 127

so  I installed gunicorn:

(h) pip install gunicorn
Collecting gunicorn
  Using cached gunicorn-19.3.0-py2.py3-none-any.whl
Installing collected packages: gunicorn
Successfully installed gunicorn-19.3.0
(h)dans-MacBook-Air:h dan$ make dev

Error: No module named paste.deploy

so I installed pasteDeploy:

pip install pasteDeploy
Collecting pasteDeploy
/Users/xxx/.virtualenvs/h/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Using cached PasteDeploy-1.5.2-py2.py3-none-any.whl
Installing collected packages: pasteDeploy
Successfully installed pasteDeploy-1.5.2

then
(h) make dev

Error: class uri 'h.server.Worker' invalid or not found:  

[Traceback (most recent call last):
  File "/Users/xxx/.virtualenvs/h/lib/python2.7/site-packages/gunicorn/util.py", line 140, in load_class
    mod = import_module('.'.join(components))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named h.server
]

Any help would be greatly appreciated,
regards.

PS when trying to setup the python dependencies for testing I am getting this error

python setup.py test
/Users/xxx/.virtualenvs/h/lib/python2.7/site-packages/setuptools/dist.py:294: UserWarning: The version specified ('0.5.1-1-gbc636bf') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.

.....

No local packages or download links found for annotator<0.15,>=0.14.2
error: Could not find suitable distribution for Requirement.parse('annotator<0.15,>=0.14.2')
(h)xxx-MacBook-Air:h dan$ python setup.py test

--- a/Makefile
+++ b/Makefile
@@ -10,15 +10,9 @@ export PIP_REQUIRE_VIRTUALENV
 
 default: deps
 
-h.egg-info: setup.py
-	@pip install -q --use-wheel -e .[dev,testing,YAML]
-	@touch h.egg-info
-
-node_modules: package.json
-	@npm install
-	@touch node_modules
-
-deps: h.egg-info node_modules
+deps:
+	pip install --use-wheel -e .[dev,testing,YAML]
+	npm install
 
 clean:
 	find . -type f -name "*.py[co]" -delete
@@ -31,7 +25,7 @@ clean:
 	rm -f h/static/styles/*.css
 	rm -f .coverage
 
-dev: deps
+dev:
 	@gunicorn --reload --paste conf/development.ini
 
 test:

